services/tracks/src/services/trackService.js

```diff
@@ -3,8 +3,26 @@
 const { Tracks } = require('../db/models/Tracks');
 const { v4: uuid } = require('uuid');
 const { axiosClient } = require('../config/axios.config');
+const { Kafka } = require('kafkajs');
+
+const kafka = new Kafka({
+  clientId: 'my-app',
+  brokers: ['kafka:9092'],
+})
 
 class TrackServices {
```

Comment:
If it is possible to rename it in singular? I suggest TrackService would look more logical